guides/gpt5_prompting_guide_pt-br.md

```
@@ -0,0 +1,519 @@
+
+### Esforço de raciocínio
+
+Fornecemos um parâmetro reasoning_effort para controlar o quão duro o modelo pensa e o quão disposto ele chama ferramentas; o padrão é médio, mas você deve escalar para cima ou para baixo dependendo da dificuldade de sua tarefa. Para tarefas complexas e multi-etapas, recomendamos raciocínio mais alto para garantir as melhores saídas possíveis. Além disso, observamos desempenho de pico quando tarefas distintas e separáveis são divididas em vários turnos de agente, com um turno para cada tarefa.
+
+### Reutilizando contexto de raciocínio com a API Responses
+
+Recomendamos fortemente usar a API Responses ao usar GPT-5 para desbloquear fluxos agente aprimorados, custos mais baixos e uso de tokens mais eficiente em suas aplicações.
+
+Vimos melhorias estatisticamente significativas nas avaliações ao usar a API Responses em comparação com Completions de Chat—por exemplo, observamos aumentos na pontuação Tau-Bench Retail de 73.9% para 78.2% apenas mudando para a API Responses e incluindo previous_response_id para passar itens de raciocínio anteriores para solicitações subsequentes. Isso permite que o modelo se refira a seus rastros de raciocínio anteriores, conservando tokens de CoT e eliminando a necessidade de reconstruir um plano do zero após cada chamada de ferramenta, melhorando latência e desempenho - este recurso está disponível para todos os usuários da API Responses, incluindo organizações ZDR.
+
+## Maximizando o desempenho de codificação, do planejamento à execução
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)